scripts/k8s.io/code-generator/generate-groups.sh

```diff
@@ -65,42 +65,52 @@ for GVs in ${GROUPS_WITH_VERSIONS}; do
 
   # enumerate versions
   for V in ${Vs//,/ }; do
+    V=$(echo "$V" | tr -d '\n\r')
     FQ_APIS+=("${APIS_PKG}/${G}/${V}")
   done
 done
 
 if [ "${GENS}" = "all" ] || grep -qw "deepcopy" <<<"${GENS}"; then
   echo "Generating deepcopy funcs"
   "${gobin}/deepcopy-gen" \
-      --input-dirs "$(codegen::join , "${FQ_APIS[@]}")" \
       -O zz_generated.deepcopy \
+      "$(codegen::join , "${FQ_APIS[@]}")" \
       "$@"
 fi
 
 if [ "${GENS}" = "all" ] || grep -qw "client" <<<"${GENS}"; then
   echo "Generating clientset for ${GROUPS_WITH_VERSIONS} at ${OUTPUT_PKG}/${CLIENTSET_PKG_NAME:-clientset}"
+  TEMP_DIR=$(mktemp -d)
+  trap 'rm -rf "${TEMP_DIR}"' EXIT
```

Comment:
--output-dir became a required param with the upgraded version - this just creates a temporary dir that's later deleted to use for that arg.